### DISCUSSION TITLE
Websockets, socket.io and graphql, ....

### DISCUSSION THREAD

**Discussion by ghost** [serverpod/serverpod#404] [Q&A] [answered]
Hello,

It would be wonderful if it supports websockets, socket.io and graphql, and the ORM supports other databases, mysql, mariadb, mongodb, sqlite, ...
Thanks.

Regards,

**Comment by vlidholt** [comment, marked as answer]
I'm converting this to a discussion as it is not regarding a single feature.

There is support for websockets. You can find the documentation here:
https://docs.serverpod.dev/concepts/streams

For the ORM suggestions there are already issues filed and it's something we are going to look at in the future!

**Comment by ghost** [comment]
Thanks Viktor. 
Indeed, there are many things, it is something to gradually include in the future...

**Comment by AristideVB** [comment]
Really looking forward for GraphQL with ServerPod 😃🚀
